src/components/categories/feature/Countdown.js

```diff
@@ -0,0 +1,68 @@
+import React,{useRef, useReducer, useState,useEffect} from 'react'
+import shoeImg from '../../../img/shoes.jpg'
+
+const Countdown = () => {
```

Comment:
Just a suggestion- you can abstract this code and create a custom hook for better reusability.